bundles/org.openhab.binding.nobohub/src/main/resources/OH-INF/i18n/nobohub.properties

```diff
@@ -1,57 +1,47 @@
 thing-type.config.nobohub.nobohub.keepaliveInterval.label = Polling interval
-thing-type.config.nobohub.nobohub.keepaliveInterval.description = Polling interval (seconds). Default: 14.
+thing-type.config.nobohub.nobohub.keepaliveInterval.description = Polling interval (seconds). Default: 14
 thing-type.config.nobohub.nobohub.serialNumber.label = Serial Number
-thing-type.config.nobohub.nobohub.serialNumber.description = Serial number of the Nob� hub (12 numbers, no spaces)
+thing-type.config.nobohub.nobohub.serialNumber.description = Serial number of the Nobo hub (12 numbers, no spaces)
 thing-type.config.nobohub.zone.id.label = Id
 thing-type.config.nobohub.zone.id.description = Id of the Zone
 
 # channel types
 
 channel-type.nobohub.activeOverrideName-channel-type.label = Active Override
-channel-type.nobohub.activeOverrideName-channel-type.description = Mode of active override, using one of the predefined states supported
+channel-type.nobohub.activeOverrideName-channel-type.description = Name of active override, using one of the predefined states supported
 channel-type.nobohub.activeOverrideName-channel-type.state.option.NORMAL = Normal
 channel-type.nobohub.activeOverrideName-channel-type.state.option.COMFORT = Comfort
 channel-type.nobohub.activeOverrideName-channel-type.state.option.ECO = Eco
 channel-type.nobohub.activeOverrideName-channel-type.state.option.Away = Away
 channel-type.nobohub.activeWeekProfile-channel-type.label = Active Week Profile Id
-channel-type.nobohub.activeWeekProfile-channel-type.description = Id of the active week profile, set via the Nob� app
+channel-type.nobohub.activeWeekProfile-channel-type.description = Id of the active week profile, set via the Nobo app
 channel-type.nobohub.activeWeekProfileName-channel-type.label = Active Week Profile Name
-channel-type.nobohub.activeWeekProfileName-channel-type.description = Name of the active week profile, set via the Nob� app
+channel-type.nobohub.activeWeekProfileName-channel-type.description = Name of the active week profile, set via the Nobo app
 channel-type.nobohub.comfort-temperature-channel-type.label = Comfort Temperature
 channel-type.nobohub.comfort-temperature-channel-type.description = The preferred Comfort temperature level set on the heater or in the binding
 channel-type.nobohub.eco-temperature-channel-type.label = Eco Temperature
 channel-type.nobohub.eco-temperature-channel-type.description = The preferred Eco temperature level set on the heater or in the binding
 channel-type.nobohub.temperature-channel-type.label = Current Temperature
 channel-type.nobohub.temperature-channel-type.description = The current temperature from a device that supports reporting temperatures
 channel-type.nobohub.weekProfiles-channel-type.label = Week Profiles
-channel-type.nobohub.weekProfiles-channel-type.description = Name of the active week profile, set via the Nob� app
-
-# User Messages
-message.missing.serial = Missing serial number in configuration
```

Comment:
These are still used, just i18n tool not generating them.

Reply:
Re-added.